api/envoy/config/trace/v2/trace.proto

```diff
@@ -83,6 +84,28 @@ message ZipkinConfig {
   // Determines whether client and server spans will shared the same span id.
   // The default value is true.
   google.protobuf.BoolValue shared_span_context = 4;
+
+  // Available Zipkin collector endpoint versions.
+  enum CollectorEndpointVersion {
+    // If not set, it fallbacks to ``HTTP_JSON_V1``.
+    NOT_SET = 0 [deprecated = true];
```

Comment:
I'm confused why the unset value is appearing, only to be immediately deprecated?

Reply:
You're right this is not needed. Was also confused that whether we need an explicit "not set" or not. Will update this to just default to `HTTP_JSON_V1` as `0`.

Reply:
This is updated. Thank you.